app/controllers/DownloadRequestSuccessController.scala

```diff
@@ -40,8 +40,13 @@ class DownloadRequestSuccessController @Inject() (
 
   def onPageLoad: Action[AnyContent] = (identify andThen profileAuth andThen getData andThen requireData).async {
     implicit request =>
-      downloadDataConnector.getEmail(request.eori).map { email =>
-        Ok(view(email.address))
+      downloadDataConnector.getEmail(request.eori).map {
+        case Some(email) => Ok(view(email.address))
+        case _           =>
```

Comment:
ahh so does this reroute to index? maybe this should be a continue JR that reroutes to the index because then that is starting them on the journey of verifying email again

Reply:
Referencing the test, this is what logErrorsAndContinue does already :)

```
val redirectUrl = Some(RedirectUrl(routes.IndexController.onPageLoad.url))

status(result) mustEqual SEE_OTHER
redirectLocation(result).value mustEqual routes.JourneyRecoveryController.onPageLoad(redirectUrl).url
```